### ISSUE TITLE
Accessibility issue: Document should have one main landmark on /az_core/mobile_nav_callback/menu_link_content%3A8afba514-09f5-466a-8288-78466ece8bb7

### ISSUE THREAD

**Issue by accesswatch** [accesswatch/qs-proposal#66]
## What
An accessibility scan flagged the element `<html>` because document should have one main landmark. Learn more about why this was flagged by visiting https://dequeuniversity.com/rules/axe/4.11/landmark-one-main?application=playwright.

[View screenshot](https://github.com/accesswatch/qs-proposal/blob/gh-cache/.screenshots/89271f37-e43b-4a9f-85ab-7e211ce40bb9.png)

To fix this, ensure the document has a main landmark.

Specifically:

Fix all of the following:
-   Document does not have a main landmark

## Acceptance Criteria
- [ ] The specific violation reported in this issue is no longer reproducible.
- [ ] The fix MUST meet WCAG 2.1 guidelines OR the accessibility standards specified by the repository or organization.
- [ ] A test SHOULD be added to ensure this specific violation does not regress.
- [ ] This PR MUST NOT introduce any new accessibility issues or regressions.
